app/src/main/kotlin/at/bitfire/davdroid/ui/intro/OpenSourceFragment.kt

```diff
@@ -46,11 +48,7 @@ class OpenSourceFragment: Fragment() {
     }
 
 
-    @HiltViewModel
-    class Model @Inject constructor(
-        @ApplicationContext val context: Context,
-        val settings: SettingsManager
-    ): ViewModel() {
+    class Model(val settings: SettingsManager): ViewModel() {
```

Comment:
Opening the open-source fragment still crashes, too.

```
21:23:08.109  E  FATAL EXCEPTION: main
                 Process: at.bitfire.davdroid, PID: 5882
                 java.lang.RuntimeException: Cannot create an instance of class at.bitfire.davdroid.ui.intro.OpenSourceFragment$Model
                 	at androidx.lifecycle.ViewModelProvider$NewInstanceFactory.create(ViewModelProvider.kt:204)
                 	at androidx.lifecycle.ViewModelProvider$AndroidViewModelFactory.create(ViewModelProvider.kt:324)
                 	at androidx.lifecycle.ViewModelProvider$AndroidViewModelFactory.create(ViewModelProvider.kt:306)
                 	at androidx.lifecycle.ViewModelProvider$AndroidViewModelFactory.create(ViewModelProvider.kt:280)
                 	at androidx.lifecycle.SavedStateViewModelFactory.create(SavedStateViewModelFactory.kt:128)
                 	at dagger.hilt.android.internal.lifecycle.HiltViewModelFactory.create(HiltViewModelFactory.java:116)
```

Reply:
🤔 It was working for me, I'll check again